src/plugins/search/plugin.search.js

```diff
@@ -269,22 +272,32 @@ BookReader.prototype.cancelSearchRequest = function () {
  */
 
 /**
- * Search Results return handler
+ * Attach some fields to search inside results
  * @param {SearchInsideResults} results
- * @param {object} options
- * @param {boolean} options.goToFirstResult
+ * @param {(pageNum: LeafNum) => PageNumString} displayPageNumberFn
  */
-BookReader.prototype.BRSearchCallback = function(results, options) {
+export function marshallSearchResults(results, displayPageNumberFn) {
   // Attach matchIndex to a few things to make it easier to identify
   // an active/selected match
   for (const [index, match] of results.matches.entries()) {
     match.matchIndex = index;
+    match.displayPageNumber = displayPageNumberFn(match.par[0].page);
```

Comment:
ok. I was hoping that the service would return it with the result sooner than later 😅